packages/mongo/collection.js

```diff
@@ -674,6 +674,13 @@ Object.assign(Mongo.Collection.prototype, {
     self._collection._ensureIndex(index, options);
   },
 
+  _createIndex(index, options) {
```

Comment:
As this is a public API we shouldn't have `_` before the name, I would go with `createIndex`